share/github-backup-utils/ghe-restore-repositories-dgit

```diff
@@ -55,15 +57,65 @@ echo "$config" > "$config_file"
 
 opts="$GHE_EXTRA_SSH_OPTS -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o PasswordAuthentication=no"
 
+# Remote sync-in-progress file location. When this file exists, Git GC
+# operations are disabled on the GitHub instance.
+sync_in_progress_file="$GHE_REMOTE_DATA_USER_DIR/repositories/.sync_in_progress"
+
 cleanup() {
   for pid in $(jobs -p); do
     kill -KILL $pid > /dev/null 2>&1 || true
   done
+  for hostname in $hostnames; do
+    ghe-ssh -F $config_file "$hostname:$port" -- "sudo rm -f '$sync_in_progress_file'"
+  done
   rm -rf $config_file ssh_routes_in ssh_routes_out ssh_finalize_in ssh_finalize_out
 }
 
 trap 'cleanup' INT TERM EXIT
 
+wait_pids=""
+for hostname in $hostnames; do
```

Comment:
I believe this will match all the servers in the cluster, not only git servers. Is that the intention?

Running something like:

```
ghe-cluster-each -r git -p
```

will return the list of git servers only. Same applies when cleaning up the `sync_in_progress_file`.